cms/templates/js/checklist.underscore

```diff
@@ -7,7 +7,7 @@
     <% } %>
     id="<%= 'course-checklist' + checklistIndex %>">
     <span class="viz viz-checklist-status"><span class="viz value viz-checklist-status-value" style="width: <%= percentChecked %>%;">
-        <%= _.template(gettext("{number}% of checklists completed"), {number: '<span class="int">' + percentChecked + '</span>'}, {interpolate: /\{(.+?)\}/g}) %>
+        <%= _.template(gettext("{number}{percent_sign} of checklists completed"), {number: '<span class="int">' + percentChecked + '</span>', percent_sign: gettext('%')}, {interpolate: /\{(.+?)\}/g}) %>
```

Comment:
Transifex treats the `% o` here as a special pattern like `%s` in C language and requires each translation keep the sequence `% o`, so I think it is better to remove the raw `%` from here.